index.html

```diff
@@ -2,13 +2,196 @@
   <title>Web Summit</title>
 </head>
 <body>
-  
+  <header class="header">
+    <i id="menu" class="fa-solid fa-bars"></i>
+
+    <div id="openNav" class="openNav">
+      <i id="close" class="fa-solid fa-close closeIcon"></i>
+
+      <ul>
+          <li>
+            <a class="item" href="./src/pages/about.html">
+              About
+            </a>
+          </li>
+          <li >
+              <a class="item" href="#About">
+              Program
+            </a>
+          </li>
+          <li >
+              <a class="item" href="#Contact">
+              Join
+            </a>
+          </li>
+          <li >
+              <a class="item" href="#Contact">
+              Sponsor
+            </a>
+          </li>
+          <li >
+              <a class="item" href="#Contact">
+              News
+            </a>
+          </li>
+          <li >
+            <a class="item item--button" href="#Contact">
+            Campaign
+          </a>
+        </li>
+      </ul>
+  </div>
+    <nav class="navbar">
+      <a href="./index.html" class="companyLogo">
+        <img alt="WebSummit_Logo" src="./src/img/download.png" />
+      </a>
+      <ul>
+        <li>
+          <a class="item" href="./src/pages/about.html">
+            About
+          </a>
+        </li>
+        <li >
+            <a class="item" href="./src/pages/about.html">
+            Program
+          </a>
+        </li>
+        <li >
+            <a class="item" href="#Contact">
+            Join
+          </a>
+        </li>
+        <li >
+            <a class="item" href="#Contact">
+            Sponsor
+          </a>
+        </li>
+        <li >
+            <a class="item" href="#Contact">
+            News
+          </a>
+        </li>
+        <li >
+            <a class="item item--button" href="#Contact">
+            Campaign
+          </a>
+        </li>
+      </ul>
+  </nav>    
+  <div class="header--content">
+    <div class="header--content--title">
+      <p class="header--content--title--greet">
+        "Hello! Sharing world"
+      </p>
+      <h1 class="header--content--title--main">“The best technology conference <br> on the planet”</h1>
+    </div>
+    <div class="header--content--main">
+    <div class="header--content--text">
+      <p class="header--content--text--paragraph">
+        Web Summit 2022 will bring together 70,000+ people, and the companies redefining the tech industry.
+      </p>
+    </div>
+    <div class="header--content-location">
+    <h2 class="header--content--time">Join us in Lisbon this November.</h2>
+    <p class="header--content--address">@ National Museum of Lisbon, ARt Center Nabi and more</p>
+    </div>
+    </div>
+
+  </div>
+
+  </header>
+
+  <main>
+    <section class="programs">
+      <div class="programFirst">
+      <h3 class="programs--title">Main Program</h3>
+      <div class="borderLine"></div>
```

Comment:
- [Optional] This line is not in the center of the `Main Program `header as it is on the original design provided to you.